net-misc/seafile-client/seafile-client-9.0.12.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Upstream is moving tags repeatedly, then we use commit hash.
+RELEASE_COMMIT="93e0eb97550999672f001216f007681157f893b5"
+
+inherit xdg cmake
+
+DESCRIPTION="Seafile desktop client"
+HOMEPAGE="https://www.seafile.com/ https://github.com/haiwen/seafile-client/"
+SRC_URI="https://github.com/haiwen/${PN}/archive/${RELEASE_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${RELEASE_COMMIT}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="shibboleth"
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-libs/jansson:=
+	dev-libs/openssl:=
+	dev-qt/qtbase:6[dbus,gui,network,widgets]
+	dev-qt/qtwebengine:6[widgets]
```

Comment:
```
	dev-qt/qtwebengine:6[widgets]
```
half-off-topic, but could you recheck that ?
Builds with success (Wreturn-type however) with; 
```
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -157,7 +155,7 @@
 ENDIF()

 IF(QT_VERSION_MAJOR EQUAL 6)
-    SET(USE_QT_LIBRARIES Core Gui Widgets LinguistTools Network Core5Compat WebEngineCore WebEngineWidgets)
+    SET(USE_QT_LIBRARIES Core Gui Widgets LinguistTools Network Core5Compat)
 ELSE()
     SET(USE_QT_LIBRARIES Core Gui Widgets LinguistTools Network)
 ENDIF()
```

After a quick glance, I find nothing that justifies that in the PR introducing Qt6.
https://github.com/haiwen/seafile-client/pull/1391/files

Reply:
@clan Can you confirm that ? If so, could you please update?
Adding qtwebengine as an option is always a good move.

Reply:
That was the case before qt6 port:
https://github.com/gentoo/gentoo/blob/972207cb3cae927bb7089337ed56521044780c9f/net-misc/seafile-client/seafile-client-9.0.3.ebuild#L35

Reply:
Thanks. Could this be added in this PR for merging ?